src/docs/ftw-styling/how-to-add-static-pages-in-ftw/index.md

```diff
@@ -220,18 +220,7 @@ Here's an example what your AboutPage.module.css file could look like:
 }
```

Comment:
This is a code-splitted page: add import for customMediaQueries:
`@import '../../styles/customMediaQueries.css';`
https://deploy-preview-395--sharetribe-flex-docs-site.netlify.app/docs/ftw-routing/how-code-splitting-works-in-ftw/#css-chunk-changes